meteor/sofie-server/business-logic/services/core-meteor-integration/sofie-core-meteor-rundown-service.ts

```diff
@@ -0,0 +1,87 @@
+import { RundownService } from '../interfaces/rundown-service'
+import { ServerClientAPI } from '../../../../server/api/client'
+import { Meteor } from 'meteor/meteor'
+import { MethodContextAPI } from '../../../../lib/api/methods'
+import { StudioJobs } from '@sofie-automation/corelib/dist/worker/studio'
+import { RundownPlaylistId } from '@sofie-automation/shared-lib/dist/core/model/Ids'
+
+export class SofieCoreMeteorRundownService implements RundownService {
+	public async activateRundown(_rundownId: string): Promise<void> {
+		throw new Error('Not implemented exception')
+	}
+
+	public async takeNext(rundownId: string): Promise<void> {
+		const rundownPlaylistId: RundownPlaylistId = rundownId as unknown as RundownPlaylistId
+		await ServerClientAPI.runUserActionInLogForPlaylistOnWorker(
+			this.getMeteorMethodContext(),
+			'TakeEvent',
+			new Date().getTime(),
+			rundownPlaylistId,
+			() => {
+				// Do nothing
+			},
+			StudioJobs.TakeNextPart,
+			{
+				playlistId: rundownPlaylistId,
+				fromPartInstanceId: null,
+			}
+		)
+	}
+
+	private getMeteorMethodContext(): MethodContextAPI {
+		return {
+			userId: null,
+			connection: this.createMeteorConnection(),
+			isSimulation: false,
+			setUserId: () => {
+				// Do nothing
+			},
+			unblock: () => {
+				// Do nothing
+			},
+		}
+	}
+
+	private createMeteorConnection(): Meteor.Connection {
+		return {
+			id: 'randomId',
```

Comment:
Not so random `randomId`. 😄 

Should it remain like this or, be updated to some actual random value?

Reply:
This file is going to be deleted. This was a proof of concept to call meteor methods, which is something we don't want to be doing going forward.